docs/usage/configuration-options.md

```diff
@@ -2200,22 +2200,64 @@ e.g.
 
 The `postUpgradeTasks` configuration consists of three fields:
 
-### commands
+### commands (postUpgradeTasks)
```

Comment:
I'm not sure if the docs generation works when title doesn't equal the config option.

@rarkins maybe i need to update it before to allow title suffix

Reply:
I think it's a markdown issue if they are multiple identical title (`commands` from `preUpgradeTasks` and from `postUpgradeTasks`) so it's for this a have added suffix.

Reply:
I think the headings must match the config option name as defined in the source code. I recently got shouted at by the docs tests for doing it wrong. 😄